### ISSUE TITLE
Runtime exception: `Ribbon resource DLL exists but could not be loaded.`

### ISSUE THREAD

**Issue by tajbender** [SunburstApps/Sunburst.WindowsForms.Ribbon#1]
Hello,

I tried to build a simple demo project, an empty form containing an empty ribbon (...also tried to modify an existing project that makes use of the original RibbonLib without success, too).

When running the project I get an exception while the form is created:

`Ribbon resource DLL exists but could not be loaded.`

Environment:

- Visual Studio 2017 Community Edition
- Windows 10 (German)

What am I doing wrong? Any help would be highly appreciated!

Thanks in advance,
tajbender

**Comment by wjk** [comment]
My fork has changed how the ribbon resource DLL is loaded as compared to how upstream RibbonLib does it. This in turn affects how you reference the ribbon from the designer. For the framework to function, you need to specify _only the base filename of the XML file, without any extension and with no namespace_, If you add ".ribbon" to the end or a namespace to the beginning, the wrong filename will be constructed and the DLL will not be found.

In your case, you would need to change the ResourceName property to be say `RibbonMarkup`. I could probably fix the exception message to be more accurate.

Also, just FYI, for some reason the zipfile you sent me is missing the PNG images referenced by the ribbon markup, so I could not fully test the solution listed above (I could only build an empty ribbon). Hope this helps!

**Comment by tajbender** [comment]
Hi, William,

thanks for your quick reply. I'm sorry, but it's been a while a was working with the Ribbon but that didn't do the trick.

 I've added a new Zip-File cause I stripped the Images to save Webspace :)

Thank you so much,
regards,
tajbender

[MySunburstRibbonDemo.zip](https://github.com/SunburstApps/Sunburst.WindowsForms.Ribbon/files/2029538/MySunburstRibbonDemo.zip)

**Comment by wjk** [comment]
You still included the namespace in the resource name. As I said above, you must specify only the filename, without any namespace or extension. If you change the property to say exactly `RibbonMarkup`, it will work.

**Comment by tajbender** [comment]
Hi, William,

I'm sooooo sorry. Sorry for stealing your time, I was pretty sure I checked this double time, however, among the blind even the one-eyed is king...

You're certainly absolutely right, now it works...
Thank you so much, and feel credited in my apps!

BTW, any hint for a free GUI-Builder for those XML-files? I searched I while ago but didn't find one, and editing them manually isn't as much fun as it may sound :)

Again, thank you very much for spending all your time. 🥇 

Regards,
tajbender
